cpp/ps0002/p0056.2.cpp

```diff
@@ -4,24 +4,26 @@ using namespace std;
 
 int main()
 {
-    int home  20 ;
+    unsigned short residents = 20; // 0.25/0.75
 
-    int  pharmase   3;
+    unsigned short alternatives = 3; // 0.25/0.75
 
-    double  progrem   200;
+    unsigned short team_members = 200; // 0.25/0.75
 
-    float bekary    5.5;
+    float pie = 5.5; // 0.75/0.75
 
-    char a;
+    char first_letter_of_name = 'a'; // 0.25/0.75
 
-    double china    100000000;
+    unsigned int chinese_births = 100000000; // 0.75/0.75
 
-    float
+    long doube division = 56546.65477; // 0/0.75
 
-    double fatora    12462422 ;
+    double fatora = 12462422; // 0.75/0.75
 
-    double age ;
+    unsigned int age = 435687643; // 0.75/0.75
 
+    // total 4/6.75

```

Comment:
- [ ] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [ ] using correct literals and notations for assigning initial values to the variables